vm-image-spec.yaml

```diff
@@ -272,6 +272,19 @@ files:
               else (pg_current_wal_lsn() - '0/0')::FLOAT8
             end as lsn;
 
+      - metric_name: receive_lsn
+        type: gauge
+        help: 'Returns the last write-ahead log location that has been received and synced to disk by streaming replication'
```

Comment:
I wonder what is the use-case you have in mind for using current_lsn and receive_lsn? I ask because absolute values are usually hard to reason about, but I don't have any objections, just curious

Reply:
Plotting receive LSN against replay LSN for computes. It would have been useful in the last replication lag investigation to see that replay LSN was not advancing while receive LSN continued on.

Reply:
Makes sense, yes. I just thought that we might want to expose the delta like `compute_replay_lag_bytes` right away. But having two distinct metrics is probably fine too, we can always subtract them